galsim/phase_screens.py

```diff
@@ -448,11 +455,18 @@ def _zern_rho_coefs(n, m):
     A[n-2*kmax] = val
     return A
 
-
-def _zern_coef_array(n, m, eps=0., shape=None, annular=False):
+def _zern_coef_array(n, m, eps, shape, annular):
     """Assemble coefficient array for evaluating Zernike (n, m) as the real part of a
     bivariate polynomial in abs(rho)^2 and rho, where rho is a complex array indicating position on
     a unit disc.
+
+    @param n        Zernike radial coefficient
+    @param m        Zernike azimuthal coefficient
+    @param eps      Linear obscuration fraction.
```

Comment:
I think the annular Zernike paper calls this parameter epsilon, so at some point in the chain it probably makes sense to switch the name to eps, since it's probably clearer at that point for comparison to the paper formulae.  But I switched it here to obscuration, since that is what we've been calling it for most of these functions.